packages/codemod/test/__snapshots__/transforms.spec.ts.snap

```diff
@@ -42,3 +42,8 @@ function Modal() {
 
 export default Modal;"
 `;
+
+exports[`testing transform "rename-mod-css-to-module-css" transforms correctly 1`] = `
+"// eslint-disable-next-line import/extensions, import/no-unresolved, no-unused-vars
+import styles from \\"./styles.module.css\\";"
```

Comment:
nit: It seems the import is changed from single quotes to double quotes. Maybe we could try to keep single quotes.

Reply:
This should do the trick: https://github.com/commercetools/merchant-center-application-kit/pull/2693/commits/3c0251ae5069dfb92da882ac9696a24b8ec1bec1